frontend/src/scenes/annotations/Annotations.tsx

```diff
@@ -0,0 +1,152 @@
+import { organizationLogic } from 'scenes/organizationLogic'
+import { AnnotationModal } from './AnnotationModal'
+
+export const scene: SceneExport = {
+    component: Annotations,
+    logic: annotationsPageLogic,
+}
+
+export function Annotations(): JSX.Element {
+    const { currentTeam } = useValues(teamLogic)
+    const { currentOrganization } = useValues(organizationLogic)
+    const { annotations, annotationsLoading, next, loadingNext } = useValues(annotationsPageLogic)
+    const { loadAnnotationsNext, openModalToCreateAnnotation, openModalToEditAnnotation } =
+        useActions(annotationsPageLogic)
+
+    const columns: LemonTableColumns<AnnotationType> = [
+        {
+            title: 'Annotation',
+            key: 'annotation',
+            width: '30%',
+            render: function RenderAnnotation(_, annotation: AnnotationType): JSX.Element {
+                return <div className="ph-no-capture">{annotation.content}</div>
+            },
+        },
+        {
+            title: 'Date and time',
+            dataIndex: 'date_marker',
+            render: function RenderDateMarker(_, annotation: AnnotationType): string {
+                // Format marker. Minute precision is used, because that's as detailed as our graphs can be
+                return dayjs(annotation.date_marker).format(ANNOTATION_DAYJS_FORMAT)
+            },
+            sorter: (a, b) => dayjs(a.date_marker).diff(b.date_marker),
+        },
+        {
+            title: 'Scope',
+            key: 'scope',
+            render: function RenderType(_, annotation: AnnotationType): JSX.Element {
+                const scopeName = annotationScopeToName[annotation.scope]
+                const tooltip =
+                    annotation.scope === AnnotationScope.Insight
+                        ? `This annotation only applies to the "${annotation.insight_name}" insight`
+                        : annotation.scope === AnnotationScope.Project
+                        ? `This annotation applies to all insights in the ${currentTeam?.name} project`
+                        : `This annotation applies to all insights in the ${currentOrganization?.name} organization`
+                return (
+                    <Tooltip title={tooltip} placement="right">
+                        <LemonTag>
+                            {annotation.scope === AnnotationScope.Insight ? (
+                                <Link
+                                    to={urls.insightView(annotation.insight_short_id as InsightShortId)}
+                                    className="flex items-center"
+                                    target="_blank"
+                                >
+                                    {scopeName}
+                                    <IconOpenInNew className="ml-1" />
+                                </Link>
+                            ) : (
+                                scopeName
+                            )}
+                        </LemonTag>
```

Comment:
These `LemonTag`s are functional but look dull:
<img width="127" alt="Screen Shot 2022-08-25 at 16 56 38" src="https://user-images.githubusercontent.com/4550621/186699177-b6c58925-9ade-4cc1-9420-a24ae2d23c9d.png">
The only color options on the component right now are success/warning/error/etc. which isn't appropriate here.
Ideas for nicer distinction within the constraints of the design system? @clarkus 
BTW, we have `LemonTag`… but we also have `LemonSnack`. I can't tell which one should I use here and why, can you offer a helping hand @benjackwhite?

Reply:
I'm not sure what a lemonsnack is. For tags, I've been using the options shown at https://www.figma.com/file/Y9G24U4r04nEjIDGIEGuKI/PostHog-Design-System-One?node-id=3139%3A1218. The blue options are a good default. I'm not sure about putting linked insight inside the tag/pill. It might be more effective if we link using the insight name?

<img width="247" alt="Screen Shot 2022-08-25 at 11 04 37 AM" src="https://user-images.githubusercontent.com/254612/186737453-82d07dc7-601e-44e1-a598-106f22ce4e44.png">

Reply:
Hmm, so we don't really have this in code – we have `LemonTag`, which looks like in the screenshots above – it's the right size but looks differently – and we have `LemonSnack` – which looks like what you've linked, except its font is 14px instead of 12px, and regular instead of semibold. We should make sure all the cases currently covered by those two components are sorted out in the design system as one "tag" component, and then we should be able to unify this into `LemonTag`. I think currently `LemonTag` is used for things like "BETA" tags, while `LemonSnack` for multi-select options.

Reply:
Agree we should consolidate into a single pattern. This is absolutely an area where designs are misaligned in figma - I recently worked through changes in a few areas. I'll make time to audit this and get a more complete approach put together in the design system assets.